### PR TITLE
The web.config file needs to have Windows slashes even if published o…

### PR DESCRIPTION
**Pull request by moozzyk**
…n *nix

Addresses #104

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs [hunk 1]
```diff
@@ -57,8 +57,10 @@ private static void TransformHandlers(XElement handlersElement)
 
         private static void TransformAspNetCore(XElement aspNetCoreElement, string appName, bool configureForAzure)
         {
-            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : ".", appName);
-            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @".\logs", "stdout.log");
+            // Forward slashes currently work neither in AspNetCoreModule nor in dotnet so they need to be
+            // replaced with backwards slashes when the application is published on a non-Windows machine
+            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : ".", appName).Replace("/", "\\");
+            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @".\logs", "stdout.log").Replace("/", "\\");
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs]: remove the `.log` extension.

**Comment by Tratcher** [comment]
:shipit:

**Comment by muratg** [comment]
:shipit:

File: src/dotnet-publish-iis/WebConfigTransform.cs [hunk 2]
```diff
@@ -57,8 +57,8 @@ private static void TransformHandlers(XElement handlersElement)
 
         private static void TransformHttpPlatform(XElement httpPlatformElement, string appName, bool configureForAzure)
         {
-            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : "..", appName);
-            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @"..\logs", "stdout.log");
+            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : "..", appName).Replace("/", "\\");
+            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @"..\logs", "stdout.log").Replace("/", "\\");
```
Comment (GuardRex) [inline src/dotnet-publish-iis/WebConfigTransform.cs]: Should this transform here and on second line above still have the `.log` on them?

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs [hunk 3]
```diff
@@ -57,8 +57,10 @@ private static void TransformHandlers(XElement handlersElement)
 
         private static void TransformAspNetCore(XElement aspNetCoreElement, string appName, bool configureForAzure)
         {
-            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : ".", appName);
-            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @".\logs", "stdout.log");
```
Comment (GuardRex) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs]: Should "stdout.log" still have the `.log` on it?

Comment (moozzyk) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs]: As per description of the commit: 

_AspNetCoreModule appends a timestamp followed by ".log" to log file name. Removing extension prevents from creating log files with names like stdout.log.{timestamp}.log_

Comment (GuardRex) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs]: Ah ... gotcha ... thanks.

**Pull request closed without merging**